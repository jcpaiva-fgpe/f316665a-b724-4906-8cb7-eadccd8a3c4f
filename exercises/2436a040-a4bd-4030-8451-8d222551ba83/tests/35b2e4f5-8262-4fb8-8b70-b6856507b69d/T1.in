1 1 1 1 0 0 1 0 0 0 0 0 1 0 1 1 1 0 0 0
0 0 0 0 0 0 0 1 0 0 0 0 0 0 0 1 0 0 0 0
0 1 0 0 0 0 1 0 0 0 0 0 1 0 0 1 0 0 0 0
1 0 1 0 0 0 0 0 1 1 0 0 0 0 0 0 0 0 0 1
0 0 0 0 0 1 0 0 0 0 0 0 1 0 0 0 0 0 0 1
0 0 0 0 0 1 0 0 0 0 0 1 0 1 0 0 0 0 0 1
0 0 1 0 0 0 0 1 0 0 0 0 0 0 0 0 1 0 0 1
0 0 0 0 1 0 0 1 1 1 0 0 0 0 0 0 0 0 0 0
0 0 1 0 0 0 0 1 0 0 1 0 1 0 0 1 0 1 0 0
0 0 0 0 0 0 0 0 0 0 0 1 0 0 0 1 0 1 0 0
6
9 1 9 8 5 3
1 1 1 2 1 3
8 2 9 3 9 9
1 4 7 8 1 10
10 2 9 7 8 8
9 1 9 3 8 1
