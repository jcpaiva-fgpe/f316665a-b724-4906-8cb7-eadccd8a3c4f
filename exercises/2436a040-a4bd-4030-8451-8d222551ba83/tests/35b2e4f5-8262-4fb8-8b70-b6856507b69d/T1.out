Jogador 1: H2 C1
Jogador 2: E3
Jogador 1: H1
Jogador 2: P1 E1
Jogador 1: H1
Jogador 2: S1
